pkgs/applications/version-management/git-and-tools/git/default.nix

```diff
@@ -7,6 +7,7 @@
 , withManual ? true
 , pythonSupport ? true
 , sendEmailSupport
+, darwin
```

Comment:
I added in `darwin` because I needed it to be able to use `darwin.Security` on line 47.

I can add in a flag but it'll only be useful if building on Darwin (because it looks for `darwin.Security`) and fail if building on other platforms.

Let me know if you think a flag is needed/makes sense and I'll add it in.